docs/en/pid-eaa-data-model.rst

```diff
@@ -287,25 +287,25 @@ Depending on the Digital Credential type **vct**, additional claims data MAY be
       - **Description**
       - **Reference**
     * - **given_name**
-      - [SD]. REQUIRED. Current First Name.
+      - [SD]. REQUIRED. Current First Name. (*String*)
       - Section 5.1 of `OIDC`_ and Commission Implementing Regulation `EU_2024/2977`_
     * - **family_name**
-      - [SD]. REQUIRED. Current Family Name.
+      - [SD]. REQUIRED. Current Family Name. (*String*)
       - Section 5.1 of `OIDC`_ and Commission Implementing Regulation `EU_2024/2977`_
     * - **birth_date**
-      - [SD]. REQUIRED. Date of Birth.
+      - [SD]. REQUIRED. Date of Birth. (*String, [ISO8601‑1] YYYY-MM-DD format*)
       - Commission Implementing Regulation `EU_2024/2977`_
     * - **birth_place**
-      - [SD]. REQUIRED. Place of Birth.
+      - [SD]. REQUIRED. Place of Birth. (*String*)
       - Commission Implementing Regulation `EU_2024/2977`_
     * - **nationality**
-      - [SD]. REQUIRED. One or more alpha-2 country codes as specified in ISO 3166-1.
+      - [SD]. REQUIRED. One or more alpha-2 country codes as specified in ISO 3166-1. (*Array of strings*)
```

Comment:
according to the oauth2 way to do, this "could" be a string orr an array of strings